types/node/tsconfig.json

```diff
@@ -7,7 +7,7 @@
         "module": "commonjs",
         "target": "esnext",
         "lib": [
-            "dom"
+            "es2021"
```

Comment:
We MUST maintain compability between node and libdom types as they are very often loaded together, removing this lib will remove an implicit compability check and break thousands of projects should they update.

Reply:
Ah, that makes sense.

I’m going to close this—doesn’t seem like the test changes in themselves are very useful.